README.rst

```diff
@@ -80,7 +80,7 @@ corresponding image pull tags and their release notes where applicable:
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+-------------------------------------------------------------------------------------------+
 | `main <https://github.com/cilium/cilium/commits/main>`__                             | daily      | ``quay.io/cilium/cilium-ci:latest``          | N/A                                                                                       |
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+-------------------------------------------------------------------------------------------+
-| `v1.16.0-rc.1 <https://github.com/cilium/cilium/commits/v1.16.0-rc.1>`__             | 2024-06-28 | ``quay.io/cilium/cilium:v1.16.0-rc.1``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.16.0-rc.1>`__  |
+| `v1.16.0-rc.0 <https://github.com/cilium/cilium/commits/v1.16.0-rc.0>`__             | 2024-06-17 | ``quay.io/cilium/cilium:v1.16.0-rc.0``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.16.0-rc.0>`__  |
```

Comment:
Hm. Something is not working here. Could you fix this up at least in the README? I guess the tooling is not detecting the rc.1 properly for some reason.